packages/components/src/FormField/FormField.tsx

```diff
@@ -58,12 +58,10 @@ function FormFieldInternal(props: FormFieldInternalProps) {
     clearable = "never",
     autofocus,
   } = props;
-
+  const formContext = useFormContext();
+  // If there isn't a Form Context being provided, get a form for this field.
   const { control, setValue, watch } =
-    useFormContext() != undefined
-      ? useFormContext()
-      : // If there isn't a Form Context being provided, get a form for this field.
-        useForm({ mode: "onTouched" });
+    formContext ?? useForm({ mode: "onTouched" });
```

Comment:
`shouldFocus` is the default on `useForm` https://react-hook-form.com/docs/useform#shouldFocusError

Reply:
Yes, for some reason I thought you were referencing the trigger with shouldFocus which is needed because trigger doesn't focus by default even with shouldFocusError being true.
Pushing the change now